docs/source/fabric_definition.rst

```diff
@@ -434,6 +470,38 @@ A switch matrix multiplexer is modelled by having multiple connections for the s
    # the same in compact form:
    N2BEG[0|0|0|0],[N2END3|E2END2|S2END1|LB_O]
 
+
+The ``INCLUDE`` keyword can also be used to include another list file to the current list file. For example:
+
+.. code-block:: python
+
+   INCLUDE, ../include/Base.list
+
+   N2BEG0,N2END3 # cascade and twist wire index
+   N2BEG0,E2END2 # turn from east to north
+   N2BEG0,S2END1 # U-turn
+   N2BEG0,LB_O   # route LUT B output north
+
+and the target file ``../include/Base.list`` contains the following:
+
+.. code-block:: python
+
+   N1BEG0,N2END3 
+   N1BEG0,E2END2 
+
+The base file is equivalent to:
+.. code-block:: python
+
+   N1BEG0,N2END3 
+   N1BEG0,E2END2
+   N2BEG0,N2END3 # cascade and twist wire index
+   N2BEG0,E2END2 # turn from east to north
+   N2BEG0,S2END1 # U-turn
+   N2BEG0,LB_O   # route LUT B output north
+
+The directory where the ``INCLUDE`` is relative to where the list file is located. For example if the file is located 
```

Comment:
```suggestion
The directory defined by the ``INCLUDE`` keyword is relative to where the list file is located. For example if the file is located 
```

I'm not entirely sure what the initial intention was, but this is what I understood it should mean, since the original sentence does not seem to be correct to me.